stor/s3.py

```diff
@@ -48,6 +49,8 @@ def _parse_s3_error(exc, **kwargs):
         msg += ', Key: ' + kwargs.get('Key')
 
     if http_status == 403:
+        if 'storage class' in msg and code == 'InvalidObjectState':
+            return exceptions.ObjectInColdStorageError(msg, exc)
```

Comment:
Can you add a comment pointing to S3 doc that explains where this condition came from?

Reply:
see the ObjectInColdStorageError exception for details